src/services/userService.ts

```diff
@@ -0,0 +1,152 @@
+'use server';
+import bcrypt from 'bcrypt';
+
+import connectDB from '../lib/dbConnect';
+import { stringToObjectId } from '../lib/utils';
+
+import TypeModel from '@/models/Type';
+import UserModel, { IUser } from '@/models/User';
+
+export async function getUsers() {
+    try {
+        await connectDB();
+
+        const users = await UserModel.find<IUser>({}, 'name');
```

Comment:
Do we not need ids of users?

Reply:
In mongodb, the id will be returned even you don't specify